packages/matter.js/src/mdns/MdnsBroadcaster.ts

```diff
@@ -137,7 +140,11 @@ export class MdnsBroadcaster {
                     PtrRecord(MATTER_SERVICE_QNAME, deviceMatterQname),
                     PtrRecord(fabricQname, deviceMatterQname),
                     SrvRecord(deviceMatterQname, { priority: 0, weight: 0, port: announcedNetPort, target: hostname }),
-                    TxtRecord(deviceMatterQname, ["SII=5000", "SAI=300", "T=1"]),
+                    TxtRecord(deviceMatterQname, [
+                        `SII=${sleepIdleInterval ?? 5000}`,  /* Sleepy Idle Interval */
```

Comment:
Constants for these defaults (5000 & 300) may be overkill but if you destructure operationalInstanceData above you could have `{ sleepIdleInterval = 5000, sleepActiveInterval = 300 } = operationalInstanceData` which would tighten up the code and I think do a good job of documenting the defaults on the API